frontend/src/pages/Resources/Resources.js

```diff
@@ -34,9 +40,31 @@ export default function Resources() {
             <section className="main-content">
               <section className="resource-list">
                 <SearchForm />
+                <div className="search-suggestions">
+                  <h4 class="header-description">Filters:</h4>
```

Comment:
- Can you please use the `font-size: 14px` for the "Filters:" text
- Also make sure the it is center horizontally with the filter chips